tests/unit/views/notifications/test_organization_admin_notification.py

```diff
@@ -0,0 +1,226 @@
+    assert 'owner' in org_ankhmorpork.roles_for(user_vetinari)
+
+
+@given("Vimes is an admin of the Ankh-Morpork organization")
+def given_vimes_admin(db_session, user_vimes, org_ankhmorpork, user_vetinari) -> None:
+    vimes_admin = models.OrganizationMembership(
+        user=user_vimes,
+        organization=org_ankhmorpork,
+        granted_by=user_vetinari,
+        is_owner=False,
+    )
+    db_session.add(vimes_admin)
+    assert 'admin' in org_ankhmorpork.roles_for(user_vimes)
+
+
+@when(
+    parsers.parse("Vetinari adds Ridcully with the role {role}"),
+    target_fixture='ridcully_admin',
+)
+@given(
+    parsers.parse(
+        "Ridcully is an existing admin with roles {role} of the Ankh-Morpork organization"
+    ),
+    target_fixture='ridcully_admin',
+)
+def when_vetinari_adds_ridcully(
+    db_session, user_vetinari, user_ridcully, org_ankhmorpork, role
+):
+    is_owner = 'owner' in role
+    ridcully_admin = models.OrganizationMembership(
+        user=user_ridcully,
+        organization=org_ankhmorpork,
+        granted_by=user_vetinari,
+        is_owner=is_owner,
+    )
+    db_session.add(ridcully_admin)
+    db_session.commit()
+    return ridcully_admin
+
+
+@then(
+    parsers.parse("{user} gets notified with {notification_string} about the addition")
+)
+@then(
+    parsers.parse(
+        "{user} gets notified with {notification_string} about the invitation"
+    )
+)
+@then(
+    parsers.parse(
+        "{user} gets notified with {notification_string} about the acceptance"
+    )
+)
+@then(parsers.parse("{user} gets notified with {notification_string} about the change"))
+def then_user_gets_notification(
+    user,
+    notification_string,
+    user_vimes,
+    user_ridcully,
+    user_vetinari,
+    ridcully_admin,
+) -> None:
+    user_dict = {
+        "Ridcully": user_ridcully,
+        "Vimes": user_vimes,
+        "Vetinari": user_vetinari,
+    }
+    preview = models.PreviewNotification(
+        models.OrganizationAdminMembershipNotification,
+        document=ridcully_admin.organization,
+        fragment=ridcully_admin,
+    )
+    user_notification = models.NotificationFor(preview, user_dict[user])
+    view = user_notification.views.render
+    assert (
+        view.activity_template().format(
+            actor=ridcully_admin.granted_by.fullname,
+            organization=ridcully_admin.organization.title,
+            user=ridcully_admin.user.fullname,
+        )
+        == notification_string
+    )
+
+
+@given(
+    parsers.parse(
+        "Vetinari invites Ridcully with role {role} to the Ankh-Morpork organization"
+    ),
+    target_fixture='ridcully_admin',
+)
+@when(
+    parsers.parse(
+        "Vetinari invites Ridcully with the role {role} to the Ankh-Morpork organization"
+    ),
+    target_fixture='ridcully_admin',
+)
+def when_vetinari_invites_ridcully(
+    db_session, user_vetinari, user_ridcully, org_ankhmorpork, role
+):
+    is_owner = 'owner' in role
```

Comment:
Why `in`? It's a string comparison, and you have to error if the string is an unknown value.